### PR TITLE
Stephen/getting started WIP

### PR DESCRIPTION
**Pull request by StephenFluin**
This is work to introduce a simpler getting started journey. The goal of this new tutorial is to simplify the learning journey for Angular, while creating sufficient knowledge to create Angular devs who know the basics, and have the necessary basis to learn from the rest of angular.io.

Rollout Plan:
1. Stage 1 would be launching on angular.io without navigation presence (soft launch) to validate and collect feedback
1. Stage 2 would be to replace the Getting Started with this Tutorial, and Rename the existing tour of heroes to "Tour of Heroes" in the nav.
1. Stage 3 would be to remove the Tour of Heroes from the navigation and put it somewhere else.

## PR Type
What kind of change does this PR introduce?

```
[x] Documentation content changes
```

### CODE REVIEW

File: aio/content/tutorial/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+Interpolation lets you render the contents of a property of your component as text in your HTML. 
+
+| Template                           | Data                     | Result              |
+|------------------------------------|--------------------------|---------------------|
+| `<h1>Welcome to {{siteName}}</h1>` | `siteName = 'My Store';` | Welcome to My Store |
+
+
+### [ ] Property Binding
+Following the mental model of HTML, components have state being given to them. This is accomplished by binding to the property of a component or HTML element.
```
Comment (blove) [inline aio/content/tutorial/getting-started.md]: Consider defining what component "state" is.

File: aio/content/tutorial/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+| Template                           | Data                     | Result              |
+|------------------------------------|--------------------------|---------------------|
+|`<img [src]="sourceUrl" [title]="imageTitle">`|sourceUrl = 'image.png';|title = 'An example image';|
+
+
+
+### ( ) Event Binding
+We can listen of standard HTML events, or custom events that we will create later on our components.
```
Comment (blove) [inline aio/content/tutorial/getting-started.md]: Change "We can listen of" --> "We can listen to"

File: aio/content/tutorial/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+```
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+```
+
+In the component's template `product-preview.component.html`, we will render the data we received from our `Input`.
+
+```
+<h3>{{product.name}}</h3>
+<p>{{product.price}}</p>
+<button>Buy</button>
```
Comment (blove) [inline aio/content/tutorial/getting-started.md]: Should there be a click event output binding on the button to invoke `buy.emit()`?

File: aio/content/tutorial/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+Anywhere in our app, we can now test our `app-product-preview`, and provide a list of products, binding their names and prices.
+
+```
+<app-product-preview [product]="{product: 'Pixel 2 XL', description: 'A great phone with one of the best cameras'}"></app-product-preview>
+```
+
+
+
+## Services
+Services are an important part of Angular applications. Services in Angular are a shared instance of a class that can be made available to any part of your applicatoin using the [dependency injection system](/guide/dependency-injection).
```
Comment (blove) [inline aio/content/tutorial/getting-started.md]: Fix typo: "applicatoin"

File: aio/content/tutorial/getting-started-data.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+```
+import { ReactiveFormsModule } from '@angular/forms';
+```
+
+```
+imports: [
+  BrowserModule,
+  FormsModule,
+  ...
```
Comment (blove) [inline aio/content/tutorial/getting-started-data.md]: Add `ReactiveFormsModule` to `imports` array.

File: aio/content/tutorial/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+
+|Symbol   |Name   |Example   |
+|---|---|---|
+|{{ }}   |interpolation   |`<p>Welcome to {{storeName}}</p>`   |
+|[ ]   |property binding   |`<img [src]="sourceUrl">`   |
+|( )   |event binding   |`<button (hover)="doHover()">Buy</button>`   |
+|*ngIf   |ngIf   |`<div *ngIf="products.length < 1">No products are currently available</div>`   |
+| *ngFor  |ngFor   |`<div *ngFor="let product of products">{{product.name}}</div>`   |
+
+You can see and play with each of these in our [Angular template](https://stackblitz.com/edit/getting-started?file=src%2Fapp%2Fapp.component.ts). The root component of an application is called our App Component, we'll begin working here.
```
Comment (blove) [inline aio/content/tutorial/getting-started.md]: Stackblitz contains error:

```
ERROR Error: InvalidPipeArgument: '$4.99 is not a number' for pipe 'CurrencyPipe'
```

File: aio/content/tutorial/getting-started-architecture.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  {path: 'products/:id', loadChildren: './product-details/product-details.module#ProductDetailsModule'}
+])
+```
+
+The string provided in `loadChildren` references the path of our module's TypeScript file, and then after the `#` sign, references the symbol that contains our module.
+
+When the user navigates to `products/42`, the router will automatically load an execute the code for the Lazy Loaded Product Details Module.
+
+### Child Route
+To setup our Product Details Module to act as a lazy loaded route, we must configure its router in a child configuratino.
```
Comment (blove) [inline aio/content/tutorial/getting-started-architecture.md]: Fix typo: "configuratino"

**Comment by brandonroberts** [comment]
Closing in favor of #27684

**Pull request closed without merging**